airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java

```diff
@@ -87,7 +91,14 @@ public void insertRecords(JdbcDatabase database, List<AirbyteRecordMessage> reco
         JavaBaseConstants.COLUMN_NAME_DATA,
         JavaBaseConstants.COLUMN_NAME_EMITTED_AT);
     final String recordQueryComponent = "(?, ?, ?),\n";
-    SqlOperationsUtils.insertRawRecordsInSingleQuery(insertQueryComponent, recordQueryComponent, database, records);
+    final List<List<AirbyteRecordMessage>> batches = Lists.partition(records, MAX_BATCH_SIZE);
+    batches.forEach(record -> {
+      try {
+        SqlOperationsUtils.insertRawRecordsInSingleQuery(insertQueryComponent, recordQueryComponent, database, record);
+      } catch (SQLException e) {
+        e.printStackTrace();
```

Comment:
Shouldn't we throw a RuntimeException right here?

Reply:
The function `insertRawRecordsInSingleQuery` throws a SQLException

Reply:
I think he means this suggestion below: 
```suggestion
        throw new RuntimeException(e);
```